### ISSUE TITLE
trigger/ linking issues

### ISSUE THREAD

**Issue by IceflowRE** [red-eclipse/base#207]
I found some (not normal things) issues.
First: i will speak everytime from doors but (i think) it is possible with other things which can be linked too.

1. Trigger door synchronization
http://redeclipse.net/forum/viewtopic.php?f=7&t=737

2. Trigger "link proximity on"
"link proximity off" opens a door when you are in the radius, close when not; "link proximity on" closed when you are in the radius but it doesnt open when not

4. Dead links (not an issue maybe)
If you have a door and a trigger linked, delete the trigger, make a new one (the trigger is automaticaly selected), now select the door. They get linked without performing a linking action.
If you hover over the door you can see the red linking, selecting it will delete the link. (Deleting one part should delete the link too)

3. Doors which are open are noclipped (not an issue maybe)
I dont know maybe this is intended but you can go through the door when it is open. (i mean the mapmodel)

**Comment by JeffBobbo** [comment]
In addition to point 4, doors which are in the process of opening are still clipped in their 'closed' position, meaning you can end up running into 'nothing' and sticking mines/grenades on 'nothing'.

**Comment by qreeves** [comment]
No. 4 is an engine thing, models can only have one rather simplistic bounding box, so it is either *on* or *off*. The rest sounds like a bug.

**Comment by JeffBobbo** [comment]
@qreeves Then I suggest you make it turn toggle when the animation is 50% done, rather than 100%, atm it's causing problems on a lot of maps because you can get to the door faster than they'll open.

**Comment by qreeves** [comment]
> Then I suggest you make it turn toggle when the animation is 50% done, rather than 100%

It already is.
